api/open_saves.proto

```diff
@@ -320,6 +324,26 @@ message QueryRecordsResponse {
   Hint hint = 3;
 }
 
+message GetAggregationRequest{
+  enum AggregationType {
+    AGGREGATION_UNDEFINED = 0;
```

Comment:
"UNDEFINED" should be fine because these will be prefixed with AggregationType. Currently it appears in Go as GetAggregationRequest_AGGREGATION_UNDEFINED.